src/features/Ens/MyDomains.tsx

```diff
@@ -47,7 +47,14 @@ export default function MyDomains({ domainOwnershipRecords }: MyDomainsProps) {
           {formatDateTime(unixTimestamp)}
         </RowAlignment>,
         <RowAlignment key={5} align="left">
-          <LinkOut link={`${ENS_MANAGER_URL}/name/${record.domainName}?utm_source=mycrypto`} />
+          <LinkApp
+            isExternal={true}
+            href={`${ENS_MANAGER_URL}/name/${record.domainName}?utm_source=mycrypto`}
+          >
+            <Box display={'inline-flex'} alignItems={'center'}>
```

Comment:
It does add two extra symbols to the syntax but I find it easier to work since it makes it easier to change the values, particularly with style-system responsive syntax.
Will not fix.
If the team feels strongly about this I'm happy to fix in separate PR.